pages/Free_for_Open_Source_Application_Security_Tools.md

```diff
@@ -183,9 +183,21 @@ Free tools of this type:
         dependencies in your GitHub projects. Supports: Java, .NET,
         JavaScript, Ruby, and Python. Your GitHub projects are
         automatically signed up for this service.
+  - Debricked: free for open source projects and/or smaller teams.
+     - Identifies, fixes and prevents known vulnerabilities. Read more at [https://debricked.com](https://debricked.com)
+     - [Create a free account](https://app.debricked.com/en/register)
+       
 
 Commercial tools of this type that are free for open source:
 
+ 
+   - [Debricked](https://debricked.com) - over 90% true positive rate in supported languages
+     - Identifies, fixes and prevents known vulnerabilities through automation without the need
+       to give access to your source code. Read more at [https://debricked.com](https://debricked.com)
+     - Allows for vulnerability management and license compliance in the same tool
+     - Features automated fix pull request to automatically fix vulnerabilities (currently only for javascript)
+     - Features one of the most complete [vulnerability databases](https://app.debricked.com/en/vulnerability-database) 
+     - Github version: [https://github.com/apps/debricked/](https://github.com/apps/debricked/) 
```

Comment:
Should maintain the existing alpha sorting.

Reply:
From my perspective it is in alphabetic order, am I missing something?

Reply:
C - Contrast would come first.

Reply:
Oups! Missed that :)